integration-test/common/src/test/java/com/tencent/rss/test/IntegrationTestBase.java

```diff
@@ -87,7 +87,7 @@ protected static ShuffleServerConf getShuffleServerConf() {
     serverConf.setInteger("rss.rpc.executor.size", 10);
     serverConf.setString("rss.server.hadoop.dfs.replication", "2");
     serverConf.setString("rss.server.uploader.base.path", "test");
-    serverConf.setLong("rss.server.disk.capacity", 1024L * 1024L * 1024L);
+    serverConf.setLong("rss.server.disk.capacity", 10L * 1024L * 1024L * 1024L);
```

Comment:
why this is necessary?

Reply:
Repartition data sometimes will use more than 1G.